Add regular scans with dependabot

atombrella opened the pull request:
There's already a PR from dependabot for upgrading Node Sass to a more recent version in #167. This PR enables a weekly scan. If you run `npm audit`, there are different warnings for the libraries. I didn't explore the severeness of them. https://docs.npmjs.com/cli/v8/commands/npm-audit

gstrauss commented:
I have addressed a fair bit of technical debt and updated most of the package versions.

This PR is ready to be considered, if it adds useful reports that are not already being handled by current dependabot activity in this repo.

gstrauss commented:
@janbrasna how does this fit into your more recent automation work?

atombrella commented:
> I have addressed a fair bit of technical debt and updated most of the package versions.

That's good :)

> This PR is ready to be considered, if it adds useful reports that are not already being handled by current dependabot activity in this repo.

I don't see any PRs nor commits from dependabot in this repo.  I think you actively need to enable it in order to get PRs from dependabot in the repo.

It's enabled in one of the Mozilla repositories. The description that's attached to a PR contains quite a lot of detailed information, such as the change log of the bumped version:
https://github.com/mozilla/experimenter/pull/11921

gstrauss commented:
> I don't see any PRs nor commits from dependabot in this repo.

That demonstrates that you have not looked at commits in the past couple months, nor at recent closed PRs.

> I think you actively need to enable it in order to get PRs from dependabot in the repo.

In a repo for which you have contributor privileges, there is a "Security" tab at the top of the page.   In this repo, it is currently:  "Code  Issues 15   Pull requests 5   Discussions   Actions   Security   Insights".  In the "Security" tab is

Dependabot alerts • Enabled    *(<== note: Enabled)*
Get notified when one of your dependencies has a vulnerability

and a link to
https://github.com/mozilla/ssl-config-generator/security/dependabot

which currently shows 0 Open and 78 Closed issues from dependabot

---

The above is why I posted
> This PR is ready to be considered, if it adds useful reports that are not already being handled by current dependabot activity in this repo.

gstrauss commented:
https://docs.github.com/en/code-security/getting-started/dependabot-quickstart-guide

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

This repo is already configured for dependabot security scans and notifications.

Your PR configures weekly scans for package updates.  Thank you for the suggestion, but at this time I do not wish to receive weekly emails for patch version bumps.  ssl-config-generator is a fairly simple site.  I think manually running `npm update` once a year or so should be fine, quarterly at most.
